File: android/app/src/main/java/org/mozilla/magnet/magnetapi/ApiMagnet.java
package org.mozilla.magnet.magnetapi;

import android.content.Context;

import org.mozilla.magnet.api.Api;

import java.util.HashMap;

/**
 * Created by wilsonpage on 28/10/2016.
 */

public class ApiMagnet extends Api {
    public ApiMagnet(Context context) {
        super(context);
        mount("channels", new ApiChannels(context));
        mount("subscriptions", new ApiSubscriptions(context));
        mount("metadata", new ApiMetadata(context));
    }
}

File: ios/ApiMagnet.swift
//
//  ApiMagnet.swift
//  Magnet
//
//  Created by Francisco Jordano on 01/11/2016.
//  Copyright © 2016 Mozilla. All rights reserved.
//

import Foundation

class ApiMagnet: ApiBase {
  
  override init() {
    super.init()
    mount("channels", api: ApiChannels())
    mount("subscriptions", api: ApiSubscriptions())
  }
}
